2024-12-19 19:01:14,564 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2024-12-19 19:01:14,566 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2024-12-19 19:01:24,775 - main - ERROR - Exception on /recommend/recipes [POST]
Traceback (most recent call last):
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 939, in ensure_sync
    return self.async_to_sync(func)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Tony\AppData\Local\Programs\Python\Python312\Lib\site-packages\flask\app.py", line 960, in async_to_sync
    raise RuntimeError(
RuntimeError: Install Flask with the 'async' extra in order to use async views.
2024-12-19 19:01:24,812 - werkzeug - INFO - 127.0.0.1 - - [19/Dec/2024 19:01:24] "[35m[1mPOST /recommend/recipes?token=eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJib3kiLCJpYXQiOjE3MzI1NDIwNzksImV4cCI6NDg4NjE0MjA3OX0.1B3tzMqLwTQa-wLVTpz8scEvue_pxqryMtfKMQuxncM HTTP/1.1[0m" 500 -
